doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -291,42 +303,22 @@ is_smartswitch(self):
       True
 ```
 
-get_module_dpu_data_port(self, index):
-```
-    Retrieves the DPU data port NPU-DPU association represented for the DPU index. Platforms that need to overwrite the platform.json file will use this API
-
-    This is valid only on the Switch and not on DPUs. On the DPUs this can return None
-
-    Args:
-        index: An integer, the index of the module to retrieve
-
-    Returns:
-        NPU-DPU port association: A string Ex: For index: 1 will return the dup0 port association which is "Ethernet224: Ethernet0" where the string left of ":" (Ethernet224) is the NPU port and the string right of ":" (Ethernet0) is the DPU port.
-```
 #### 3.1.3 NPU to DPU data port mapping
-platform.json of NPU/switch will show the NPU to DPU data port mapping. This will be used by services early in the system boot. 
+platform.json of NPU/switch will show the NPU to DPU data port mapping. This will be used by services early in the system boot.
 ```
-{
-    "DPUs" : [
+    "DPUS": [
         {
-        "dpu0": {
-                    "interface": {"Ethernet224": "Ethernet0"}
-        }
-        },
-        {
-        "dpu1": {
-                    "interface": {"Ethernet232": "Ethernet0"}
+            "dpu0": {
+                "interface": {"Ethernet224": "Ethernet0"}
+            },
+            "dpu1": {
+                "interface": {"Ethernet232": "Ethernet0"}
+            },
+            "dpux": {
+                "interface": {"Ethernet2xx": "Ethernet0"}
             },
-        },
-        .
-        .
-        {
-        "dpuX": {
-                    "interface": {"EthernetX": "EthernetY"}
-            }
         }
     ]
-}
 ```
 #### 3.1.4 ModuleBase class API enhancements
```

Comment:
@rameshraghupathy can you confirm if all these APIs have been defined in platform-common?

Reply:
Yes, All these APIs have been defined in platform-common